infra/multichain-dev/main.tf

```diff
@@ -83,29 +91,50 @@ module "gce-container" {
     ]
 }
 
+resource "google_service_account" "service_account" {
+  account_id   = "multichain-${var.env}"
+  display_name = "Multichain ${var.env} Account"
+}
+
+resource "google_project_iam_member" "sa-roles" {
+  for_each = toset([
+    "roles/datastore.user",
+    "roles/secretmanager.admin",
+    "roles/storage.objectAdmin",
+    "roles/iam.serviceAccountAdmin",
+    "roles/artifactregistry.reader",
+    "roles/logging.logWriter",
+  ])
+
+  role    = each.key
+  member  = "serviceAccount:${google_service_account.service_account.email}"
+  project = var.project_id
+
+}
+
 resource "google_compute_address" "internal_ips" {
   count        = length(var.node_configs)
   name         = "multichain-dev-${count.index}"
   address_type = "INTERNAL"
   address      = var.node_configs["${count.index}"].ip_address
   region       = var.region
-  subnetwork   = "projects/pagoda-shared-infrastructure/regions/us-central1/subnetworks/dev-us-central1"
+  subnetwork   = "projects/sig-shared-network/regions/europe-west1/subnetworks/dev-europe-west1"
 }
 
 module "mig_template" {
   count      = length(var.node_configs)
   source     = "../modules/mig_template"
-  network    = "projects/pagoda-shared-infrastructure/global/networks/dev"
-  subnetwork = "projects/pagoda-shared-infrastructure/regions/us-central1/subnetworks/dev-us-central1"
+  network    = "projects/sig-shared-network/global/networks/dev"
+  subnetwork = "projects/sig-shared-network/regions/europe-west1/subnetworks/dev-europe-west1"
   region     = var.region
   service_account = {
-    email  = "mpc-recovery@pagoda-discovery-platform-dev.iam.gserviceaccount.com",
+    email  = google_service_account.service_account.email,
     scopes = ["cloud-platform"]
   }
   name_prefix          = "multichain-${count.index}"
   source_image_family  = "cos-113-lts"
   source_image_project = "cos-cloud"
-  machine_type         = "n2-standard-2"
+  machine_type         = "e2-medium"
```

Comment:
is this cheaper now compared with n2-standard-2?

Reply:
Considerably yes, we could probably do even cheaper.

Reply:
I do not think this is the reason, but our dev network is slower now. Let's keep the hardware in mind.